### PR TITLE
Add log_url table to the database.

### LINKED ISSUE #448: Add log_url table to the database.
Make the following database changes:

- Add a primary key called `id` to the resource table, and change the `uuid` to be a unique index.
- Add the `log_url` table, created a foreign key from the `resource_id` field to `resource.id`.

### PR DESCRIPTION
**Pull request by ggallen**
## Which Issue(s) this Pull Request Resolves

Resolves #448

## Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by rh-hemartin** [comment]
You need to add the table to https://kubearchive.github.io/kubearchive/main/reference/database.html

**Comment by ggallen** [comment]
> You need to add the table to https://kubearchive.github.io/kubearchive/main/reference/database.html

Fixed.

File: docs/modules/ROOT/pages/reference/database.adoc [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
 [%header, cols="2m,2m,3"]
 |===
 |Name
 |Type
 |Description
 
+|id
+|id primary key
+|The ID of the Kubernetes resource used as primary key.
+
```
Comment (mafh314) [inline docs/modules/ROOT/pages/reference/database.adoc:16]: I'm sure about this change. I thought that we were using the UUID as primary key.

Comment (ggallen) [reply docs/modules/ROOT/pages/reference/database.adoc]: @mafh314, this should be fixed now (it's gone).

**Review by maruiz93 — APPROVED** [review]

**Pull request merged**